settings.gradle.kts

```diff
@@ -43,6 +43,7 @@ module("web-crawler", "./tutorials/web-crawler")
 module("media-player", "./tutorials/media-player")
 // JxBrowser logs redirection
 module("jul-logs-redirect", "./tutorials/jul-logs-redirect")
+module("taking-screenshots", "./tutorials/taking-screenshots")
```

Comment:
This tutorial is not about how to take screenshots. JxBrowser is not a screenshot tool. Please suggest a better name. Maybe html2png, html-to-png, web-page-to-png?